install_phantomjs2_debian.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# This script install PhantomJS in your Debian/Ubuntu System
+# https://gist.github.com/julionc/7476620
+#
+# This script must be run as root:
+# sudo sh install_phantomjs.sh
+#
+
+if [[ $EUID -ne 0 ]]; then
+    echo "This script must be run as root" 1>&2
+    exit 1
+fi
+
+PHANTOM_VERSION="phantomjs-2.1.1"
```

Comment:
I just looked into this, and it appears that phantomjs is using [master as their development branch](https://bitbucket.org/ariya/phantomjs/branches/), accepting features [off master](https://bitbucket.org/ariya/phantomjs/src/master/CONTRIBUTING.md?at=master&fileviewer=file-view-default). Currently master has a [failing build](https://bitbucket.org/ariya/phantomjs/commits/5d99f2a7da4820f015fb5af89be3fb66f3a868d3), I'm not really sure why. It seems using a tagged stable, with the ability to override the version variable is a more responsible solution, e.g.: `PHANTOM_VERSION="phantomjs-1.9.8"; ./install_phantomjs2_debian.sh`